src/wincng.c

```diff
@@ -394,6 +477,31 @@ _libssh2_wincng_init(void)
     if(!BCRYPT_SUCCESS(ret)) {
         _libssh2_wincng.hAlgDH = NULL;
     }
+
+#if LIBSSH2_ECDSA
+    for(unsigned int curve = 0; curve < _countof(_libssh2_wincng_ecdsa_algorithms); curve++) {
```

Comment:
inline (and in-block) declarations are C99, we prefer to keep the codebase C89.
Let's make this a block and declare on top or declare on top of existing block with guards.